### DISCUSSION TITLE
External Return Address

### DISCUSSION THREAD

**Discussion by yilakb** [ordinals/ord#1723] [Ideas]
I was wondering if it is possible to add an option where the return address could be any external taproot address when inscribing.

If not I have a suggestion for the future of the ordinal inscription process. Currently, the return transaction after inscribing ord comes back to the return address of the same wallet. However, it would be beneficial to have an option where the return address could be any external address. This way, service can inscribe ord and the receiver of the inscription could be someone else reducing network traffic.

This feature would allow for more flexibility and convenience in the ordinal inscription process, as the user would not have to transfer the ord to another wallet after the inscription. I believe this would also increase the usability of the ordinal inscription process, making it more accessible and user-friendly.

Thank you for considering this suggestion if this is technically possible and I hope you will consider incorporating it into future updates. I am available to discuss further and provide any additional information or assistance if needed.

**Comment by raphjaph** [comment]
We have an open PR for this #1536. Have a look!
